amf-validation/shared/src/main/scala/amf/plugins/features/validation/RemodValidationRunner.scala

```diff
@@ -0,0 +1,52 @@
+package amf.plugins.features.validation
+
+import amf.ProfileName
+import amf.client.remod.amfcore.plugins.validate.{AMFValidatePlugin, ValidationOptions, ValidationResult}
+import amf.core.model.document.BaseUnit
+import amf.core.validation.AMFValidationReport
+
+import scala.concurrent.{ExecutionContext, Future}
+
+trait RemodValidationRunner {
```

Comment:
liked the abstraction of the runner, however i dont find much value in the factory in this case.

Reply:
Yes, it's something legacy that was left from the first iteration. Left it there because there are plans to have parallel execution for example.

I could remove it though, it doesn't make much sense to leave it there anyway.

Reply:
Removed